lib/diaspora_federation/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diaspora.github.io/diaspora_federation/schemas/federation_entities-0.2.0-dev.json",
+  "oneOf": [
+    {"$ref": "#/definitions/comment"},
+    {"$ref": "#/definitions/like"},
+    {"$ref": "#/definitions/participation"},
+    {"$ref": "#/definitions/poll_participation"},
+    {"$ref": "#/definitions/status_message"},
+    {"$ref": "#/definitions/reshare"},
+    {"$ref": "#/definitions/profile"},
+    {"$ref": "#/definitions/location"},
+    {"$ref": "#/definitions/photo"},
+    {"$ref": "#/definitions/poll"},
+    {"$ref": "#/definitions/poll_answer"}
+  ],
+
+  "definitions": {
+    "signature": {
+      "type": "string",
+      "minLength": 30
+    },
```

Comment:
Maybe also add a type for `guid` here, to be at least 16 and at most 255 chars long?